parts/common/components.json

```diff
@@ -668,6 +669,10 @@
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes-csi/csi-node-driver-registrar",
           "latestVersion": "v2.12.0"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes-csi/csi-node-driver-registrar",
+          "latestVersion": "v2.13.0"
```

Comment:
Same question as for liveness probe:

This means we'll be caching both 2.14 and 2.15 of liveness prob:

* Do we need both of them? They are 164MB each on windows
* If the 2.12.0 is only needed while RP is rolled out to use 2.13 (due to caching) then we should use "previousLatestVersion" for 2.12 (as opposed to a separate block). This indicates to the next developer that they might be able to remove 2.12 rather than keep it.